### ISSUE TITLE
[Bug]: male/female should only map to URIs when used with `gender`

### ISSUE THREAD

**Issue by azaroth42** [project-lux/luxy#3]
### Priority Level

High

### What happened?

Fails to find concepts by name that match `male` as it gets turned into the URI for the concept by the code

### Relevant log output

```shell
>>> from luxy import Concepts
>>> res = Concepts().filter(name="male").get()
Could not find total items in response
>>> res.json
{'statusCode': 400, 'errorMessage': "Invalid search request: the 'name' term contains another term but is not allowed to."}
```

### Additional Tasks

- [ ] Task 1
- [ ] Task 2
- [ ] Task 3

**Comment by wjbmattingly** [comment]
Thanks! Fixed in 0.0.2 and will be merged into main soon.

**Comment by wjbmattingly** [comment]
Closed with #4
